src/components/SendFundsForm.vue

```diff
@@ -283,7 +284,18 @@ export default {
   },
   emits: ['send', 'error'],
   setup() {
+    const store = useStore()
+
+    const isOnline = computed(() => {
+      return store.getters['isOnline']
+    })
```

Comment:
Suggestion. Resolve if you leave as is.

```suggestion
    const isOnline = computed(() => store.getters['isOnline'])
```